modules/ztp-retrieving-the-host-firmware-settings.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ztp-deploying-far-edge-sites.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-retrieving-the-host-firmware-settings_{context}"]
+= Retrieving the host firmware settings for a managed cluster
+
+You can retrieve the host firmware settings for managed clusters.
+This is useful when you have deployed changes to the host firmware and you want to monitor that the changes and ensure that they are applied successfully.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have installed {rh-rhacm-first} and logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have provisioned a cluster that is managed by {rh-rhacm}.
+
+.Procedure
+
+. Retrieve the host firmware settings for the managed cluster.
+Run the following command:
++
+--
+[source,terminal]
+----
+$ oc get hostfirmwaresettings -n <managed_cluster_namespace> -o yaml
```

Comment:
Might be helpful to spec this as `oc get hostfirmwaresettings -n <cluster_namespace> <node_name> -o yaml` to demonstrate how to get it for just one node. In the case of SNO it's the same, but for SNO+1 or any multinode cluster you may want to be more specific.